build.gradle

```diff
@@ -180,6 +180,7 @@ dependencies {
 
     // ZipArchive dependencies used for integration tests
     zipArchive group: 'org.opensearch.plugin', name:'opensearch-ml-plugin', version: "${opensearch_build}"
+
```

Comment:
Extra line

Reply:
Is this added by spotless?

Reply:
I guess so, I run spotlessApply everytime and its passing spotless check